File: Prueba y error/ejercicios de semana santa java/four java/Library/Estudiante.java
package Library;

public class Estudiante {
    private String id, nombres, apellidos, direccion, telefono, rh;
    private Licencia licencia;

    public Estudiante(String id, String nombres, String apellidos, String direccion, String telefono, String rh, Licencia licencia) {
        this.id = id;
        this.nombres = nombres;
        this.apellidos = apellidos;
        this.direccion = direccion;
        this.telefono = telefono;
        this.rh = rh;
        this.licencia = licencia;
    }

    public String getInformacion() {
        return "Estudiante: " + nombres + " " + apellidos + "\n" +
               "Identificación: " + id + "\n" +
               "Dirección: " + direccion + "\n" +
               "Teléfono: " + telefono + "\n" +
               "RH: " + rh + "\n" +
               licencia.detallesLicencia();
    }
}

File: Prueba y error/ejercicios de semana santa php/Four php/Library/Estudiante.php
<?php

require_once 'Licencia.php';

class Estudiante {
    private string $id;
    private string $nombres;
    private string $apellidos;
    private string $direccion;
    private string $telefono;
    private string $rh;
    private Licencia $licencia;

    public function __construct($id, $nombres, $apellidos, $direccion, $telefono, $rh, Licencia $licencia) {
        $this->id = $id;
        $this->nombres = $nombres;
        $this->apellidos = $apellidos;
        $this->direccion = $direccion;
        $this->telefono = $telefono;
        $this->rh = $rh;
        $this->licencia = $licencia;
    }

    public function getInformacion(): string {
        return "=== Información del Estudiante ===\n" .
               "Nombre: {$this->nombres} {$this->apellidos}\n" .
               "Identificación: {$this->id}\n" .
               "Dirección: {$this->direccion}\n" .
               "Teléfono: {$this->telefono}\n" .
               "RH: {$this->rh}\n\n" .
               $this->licencia->detallesLicencia();
    }
}